tests/Integration/inc/Engine/License/Subscriber/displayUpgradePopin.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\Engine\License\Subscriber;
+
+use WP_Rocket\Tests\Integration\TestCase;
+
+/**
+ * @covers \WP_Rocket\Engine\License\Subscriber::display_upgrade_popin
+ *
+ * @group License
+ * @group AdminOnly
+ */
+class DisplayUpgradePopin extends TestCase {
+	private static $user;
+	private static $pricing;
+
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		$container     = apply_filters( 'rocket_container', null );
+		self::$user    = $container->get( 'user' );
+		self::$pricing = $container->get( 'pricing' );
+	}
+
```

Comment:
As these objects are static, let's restore the original property values before leaving the test class. Why? To ensure the original state is restored for other tests (existing and future).

Reply:
Updated